cmd/skaffold/app/cmd/cmd.go

```diff
@@ -85,13 +85,15 @@ func NewSkaffoldCommand(out, errOut io.Writer) *cobra.Command {
 			// These are used for command completion and send debug messages on stderr.
 			if cmd.Name() != cobra.ShellCompRequestCmd && cmd.Name() != cobra.ShellCompNoDescRequestCmd {
 				instrumentation.SetCommand(cmd.Name())
-				out := output.GetWriter(out, defaultColor, forceColors, timestamps)
+				lvl, err := logrus.ParseLevel(v)
+				if err != nil {
+					return fmt.Errorf("parsing log level: %w", err)
+				}
+				out := output.GetWriter(out, defaultColor, forceColors, timestamps, lvl)
```

Comment:
I get confused across `skaffoldWriter`, `colorableWriter`, `io.Writer` and `logrus`. 😅